src/metadata/options/default.option.ts

```diff
@@ -0,0 +1,6 @@
+export interface DefaultOption {
```

Comment:
default or common? which one is preferable?

Reply:
what about only `Option`, `Metadata` and `Arg` instead of `DefaultOption`, `DefaultMetadata` and `DefaultArg`?

Reply:
I had some progress in other refactoring and will do this in my upcoming refactor PR

Reply:
✅  Done at https://github.com/indigotech/graphql-schema-decorator/pull/34/commits/ea052196b5620d609182ccbabaa13096036c252c